examples/redis/README.md

```diff
@@ -0,0 +1,241 @@
+# Redis
+
+[Redis](https://redis.io/) is the open source, in-memory data store used by millions of developers
+as a database, cache, streaming engine and message broker.
+
+## Introduction
+
+We will be using [Redis](https://github.com/bitnami/charts/tree/main/bitnami/redis) chart that bootstraps a Redis deployment on a [Kubernetes](http://kubernetes.io) cluster using the [Helm](https://helm.sh) package manager.
+
+## Prerequisites
+
+- Kubernetes 1.20+
+- PV provisioner support in the underlying infrastructure
+- Kanister controller version 0.85.0 installed in your cluster, let's assume in Namespace `kanister`
+- Kanctl CLI installed (https://docs.kanister.io/tooling.html#install-the-tools)
+- Redis docker image. To build and push the docker image to registry, please execute [these](#build-docker-image) steps.
```

Comment:
```suggestion
- Docker CLI installed
- A docker image containing the required tools to back up Redis. The Dockerfile for the image can be found [here] (https://raw.githubusercontent.com/kanisterio/kanister/master/docker/redis-tools/Dockerfile). To build and push the docker image to your docker registry, execute [these](#build-docker-image) steps.
```
I have not formatted this correctly, we can do that in a follow-up.

Reply:
fixed